samples/matter/light_bulb/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -8,7 +8,7 @@
 CONFIG_BT_LL_SOFTDEVICE_MULTIROLE=y
 
 # nRF54L15 requires bigger stack sizes than nRF52/nRF53 families
-CONFIG_CHIP_MALLOC_SYS_HEAP_SIZE=10240
+CONFIG_CHIP_MALLOC_SYS_HEAP_SIZE=7168
```

Comment:
I wonder if we can remove this line completely and have a default values in sdk-connectedhomeip only (potentially different for nRF52/53/54).

Btw. In this PR you have changes configuration only for nRF54L15 - what about 52 and 53?

Also i believe we should run nightly on this PR before merging - as previously some of the Matter tests were failing (maybe its not the case anymore): https://github.com/nrfconnect/sdk-nrf/commit/cdcb5801a946e9969780f9d7b8d935654751c77e

Reply:
Yes, we can. We will use default configuration for all platforms - 8KB.